src/pages/invite.js

```diff
@@ -41,6 +41,24 @@ class InvitePage extends React.Component {
     });
   };
 
+  postUrl() {
+    return ('users/' + this.props.context.result.user.uid + '/invites'); 
```

Comment:
This should be fixed - an `invite` is a REST resource belonging to a `user`. When you add a new `invite` resource, you POST a new invite resource to the `invites` collection to the user.